specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2020-10-01/bms.json

```diff
@@ -0,0 +1,313 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-10-01",
+    "title": "RecoveryServicesBackupClient",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": false
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
```

Comment:
>paths [](start = 3, length = 5)

I've noticed something strange when diffing this PR with [https://github.com/Azure/azure-rest-api-specs/pull/10059](https://github.com/Azure/azure-rest-api-specs/pull/10059). There are significant differences between that PR and this PR. It seems like this PR is replacing data move and vault storage config paths with encryption config. Did I miss something here or is this intentional? If intentional, can you explain what is going on here?

Reply:
2020-10-01 has only the new APIs of CMK. The changes on vault storage config and data move are in older versions. Backup service has different jsons for different versions. The effort to have unified json as mailed is in progress, and on track for fix by end of Fe Q2.

Reply:
In these case recommended way is to create different files and name them based on the scenarios covered in the file. For e.g. you can name it as CMK.json for e.g.

Reply:
We are working on coming up with single json, thats tracked for end of Fe Q2.